pincer/commands/commands.py

```diff
@@ -705,6 +741,145 @@ async def add_commands(self, commands: List[AppCommand]):
         """
         await gather(*map(lambda cmd: self.add_command(cmd), commands))
 
+    def __build_local_commands(self):
+        """Builds the commands into the format that Discord expects. See class info
+        for the reasoning.
+        """
+        for cmd in ChatCommandHandler.register.values():
+
+            if cmd.sub_group:
+                # If a command has a sub_group, it must be nested to levels deep.
+                #
+                # command
+                #     subcommand-group
+                #         subcommand
+                #
+                # The chidlren of subcommand-group object are being set to include `cmd`
+                # If that subcommand-group object does not exist, it will be created
+                # here. The same goes for the top level command.
+                #
+                # First make sure the command exists. This command will hold the
+                # subcommand-group for `cmd`.
+
+                # `key` represents the hash value for the top level command that will
```

Comment:
```suggestion
                # The children of the subcommand-group object are being set to include `cmd`
                # If that subcommand-group object does not exist, it will be created
                # here. The same goes for the top-level command.
                #
                # First make sure the command exists. This command will hold the
                # subcommand-group for `cmd`.

                # `key` represents the hash value for the top-level command that will
```